kitsune/motidings/tests/test_unsubscribe.py

```diff
@@ -0,0 +1,44 @@
+from nose.tools import eq_
+
+from kitsune.motidings.tests import WatchFactory
+from kitsune.sumo.tests import TestCase
+from kitsune.sumo.urlresolvers import reverse
+
+
+class UnsubscribeTests(TestCase):
+    def test_view_get(self):
+        watch = WatchFactory()
+
+        url = reverse('tidings.unsubscribe', args=(watch.id,))
+        resp = self.client.get(url, follow=True)
+
+        eq_(resp.status_code, 200)
+        self.assertTemplateUsed('motidings/unsub.html')
+
+    def test_view_post_success(self):
+        watch = WatchFactory(
+            secret='ou812'
+        )
```

Comment:
Nit: This wrapping seems awkward to me. Is there a good reason to have it across 3 lines like this?